### DISCUSSION TITLE
windows10 Unable to compile

### DISCUSSION THREAD

**Discussion by getshellbypass** [rustdesk/rustdesk#527] [General]
error: failed to run custom build command for `magnum-opus v0.4.0 (https://github.com/open-trade/magnum-opus#3c3d0b86)`

Caused by:
  process didn't exit successfully: `C:\Users\user\Desktop\rustdesk-master\target\debug\build\magnum-opus-b22ba68ce3a19de0\build-script-build` (exit code: 101)
  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: NotPresent', C:\Users\user\.cargo\git\checkouts\magnum-opus-7580b098b18e2bc5\3c3d0b8\build.rs:7:50
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed

**Comment by Kingtous** [comment]
make sure vcpkg and `libyuv, libvpx, opus` is installed correctly, and set `VCPKG_ROOT` to the base folder of vcpkg.

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/dev/build/windows/
